website/src/pages/docs/guides/graphql-server-apollo-yoga-with-server-preset.mdx

```diff
@@ -140,21 +158,22 @@ We can use generated files in GraphQL API implementation:
 <Tabs items={['GraphQL Yoga','Apollo Server']}>
 
 <Tab>
+
```

Comment:
TIL the right syntax requires this extra line after `<Tab>`. Otherwise, it won't format correctly